### PR TITLE
[cu-2pbvg9e] Save pool stats as a time series

### PR DESCRIPTION
**Pull request by vimukthi-git**
## Description
- Save all entities indexed by the event that triggered an update as new entities to keep track of history

## Checklist

- [x] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [x] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2pbvg9e Funds raised aggregator RPC (subsquid)](https://app.clickup.com/t/2pbvg9e)

File: subsquid/src/pabloProcessor.ts [hunk 1]
```diff
@@ -118,8 +122,8 @@ export async function processPoolCreatedEvent(ctx: EventHandlerContext, event: P
     }
 }
 
-export function createPoolAssetId(poolId: string, assetId: bigint): string {
-    return poolId + '-' + assetId;
+export function createPoolAssetId(eventId: string, poolId: bigint, assetId: bigint): string {
+    return eventId + '-' + poolId + '-' + assetId;
```
Comment (saadjhk) [inline subsquid/src/pabloProcessor.ts:126]: @vimukthi-git how does adding an event id solve the problem of querying initial balances within the pool?

Comment (vivekvpandya) [reply subsquid/src/pabloProcessor.ts:126]: so is this eventId a timestamp?

Comment (vimukthi-git) [reply subsquid/src/pabloProcessor.ts:126]: No, it is just a unique id, that allows to store all historic versions of an entity without conflict. Timestamps already exists in the entities.

**Review by andor0 — APPROVED** [review]

**Review by saadjhk — APPROVED** [review]

File: subsquid/src/pabloProcessor.ts (hunk 1 above)
Comment (andor0) [reply subsquid/src/pabloProcessor.ts:126]: may be 
```
sha256(`${eventId}-${poolId}-${assetId}`)
```
better?

**Pull request merged**